handlers/sf-billing-account-remover/cfn.yaml

```
@@ -55,40 +67,36 @@ Resources:
           Stage: !Ref Stage
           apiAccessKeyId:
             !Sub
-            - '{{resolve:secretsmanager:sf-billing-account-remover-${Stage}:SecretString:apiAccessKeyId::${SecretsVersion}}}'
-            - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SecretsVersion]
+            - '{{resolve:secretsmanager:${Stage}/Zuora/${Account}:SecretString:apiAccessKeyId::${SecretsVersion}}}'
+            - Account: !FindInMap [StageMap, !Ref Stage, ZuoraAccount]
+              SecretsVersion: !FindInMap [StageMap, !Ref Stage, ZuoraSecretsVersion]
           apiSecretAccessKey:
             !Sub
-            - '{{resolve:secretsmanager:sf-billing-account-remover-${Stage}:SecretString:apiSecretAccessKey::${SecretsVersion}}}'
-            - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SecretsVersion]
-          authUrl:
-            !Sub
-            - '{{resolve:secretsmanager:sf-billing-account-remover-${Stage}:SecretString:authUrl::${SecretsVersion}}}'
-            - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SecretsVersion]
+            - '{{resolve:secretsmanager:${Stage}/Zuora/${Account}:SecretString:apiSecretAccessKey::${SecretsVersion}}}'
+            - Account: !FindInMap [StageMap, !Ref Stage, ZuoraAccount]
+              SecretsVersion: !FindInMap [StageMap, !Ref Stage, ZuoraSecretsVersion]
+          authUrl: !FindInMap [StageMap, !Ref Stage, SalesforceUrl]
           clientId:
             !Sub
-            - '{{resolve:secretsmanager:sf-billing-account-remover-${Stage}:SecretString:clientId::${SecretsVersion}}}'
-            - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SecretsVersion]
+            - '{{resolve:secretsmanager:${Stage}/Salesforce/MembersDataApi:SecretString:clientId::${SecretsVersion}}}'
+            - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SalesforceSecretsVersion]
           clientSecret:
             !Sub
-            - '{{resolve:secretsmanager:sf-billing-account-remover-${Stage}:SecretString:clientSecret::${SecretsVersion}}}'
-            - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SecretsVersion]
+            - '{{resolve:secretsmanager:${Stage}/Salesforce/MembersDataApi:SecretString:clientSecret::${SecretsVersion}}}'
+            - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SalesforceSecretsVersion]
           password:
             !Sub
-            - '{{resolve:secretsmanager:sf-billing-account-remover-${Stage}:SecretString:password::${SecretsVersion}}}'
-            - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SecretsVersion]
+            - '{{resolve:secretsmanager:${Stage}/Salesforce/MembersDataApi:SecretString:password::${SecretsVersion}}}'
```

A new secret has been generated that should be used for this:

```suggestion
            - '{{resolve:secretsmanager:${Stage}/Salesforce/User/MembersDataApi:SecretString:password::${SecretsVersion}}}'
```
https://eu-west-1.console.aws.amazon.com/secretsmanager/home?region=eu-west-1#/secret?name=DEV%2FSalesforce%2FUser%2FMembersDataAPI

I've implemented the '/User' section to the naming structure to differentiate ConnectedApp secrets from User secrets. The secret listed (without the 'User') should be deleted.